File: Python/Chapter 1/1.py
if __name__ == '__main__':
    print ('Sum of natural numbers divided by 3 or 5')    
    limit_value = eval(input())
    sum_value = 0

    for _ in range(3, limit_value):
        if _ % 3 == 0 or _ % 5 == 0:
            sum_value += _

    print ('Sum = ', sum_value)

File: C++/Chapter 1/1.cpp
#include <iostream>

int main(){
    std::cout<<"Sum of numbers divided by 3 and 5"<<std::endl;
    
    unsigned int limit;
    std::cin>>limit;

    unsigned long long sum = 0;
    for(unsigned int i=3; i<limit; i++){
        if (i % 3 == 0 || i % 5 == 0){
            sum += i;
        }
    }
    
    std::cout<< "Sum = " << sum << std::endl;
}
